### PR TITLE
Render reports to PDF

### LINKED ISSUE #319: generate PDF reports
and make them available for download (continuation of #158). The new report figures should have SVG plots to be included.

### PR DESCRIPTION
**Pull request by alorenzo175**
- [x] Closes #319 .
  - [x] I am familiar with the [contributing guidelines](https://solarforecastarbiter-core.readthedocs.io/en/latest/contributing.html).
  - [x] Tests added.
  - [x] Updates entries to [`docs/source/api.rst`](https://github.com/SolarArbiter/solarforecastarbiter-core/blob/master/docs/source/api.rst) for API changes.
  - [x] Adds descriptions to appropriate "what's new" file in [`docs/source/whatsnew`](https://github.com/SolarArbiter/solarforecastarbiter-core/tree/master/docs/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
  - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
  - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by alorenzo175** [comment]
an example for the current code

[test_output.pdf](https://github.com/SolarArbiter/solarforecastarbiter-core/files/4602071/test_output.pdf)

**Comment by wholmgren** [comment]
Wow! That's great.

**Comment by alorenzo175** [comment]
Right now, I've basically copied the HTML report. In the future, we'll probably want to make the two pretty different like including most plots in an appendix and condense most information in the body text/tables

File: docs/source/whatsnew/1.0.0rc1.rst [hunk 1]
```diff
@@ -25,6 +25,7 @@ Enhancements
   reports (:issue:`360`) (:pull:`437`)
 * Added support for NREL PVDAQ sites to the reference database functions.
   (:issue:`397`) (:pull:`438`)
+* Reports can now be rendered into PDF files via pdflatex (:issue:`319`) (:pull:`439`)
```
[the remark this answers is not in the document]
Comment (alorenzo175) [reply docs/source/whatsnew/1.0.0rc1.rst:28]: sure

[the remark this answers is not in the document]
Comment (alorenzo175) [reply docs/source/whatsnew/1.0.0rc1.rst:28]: easiest is for something like
image

**Pull request merged**